src/pages/user_profile/AffiliationAndRoles.js

```diff
@@ -142,67 +141,67 @@ export default function AffiliationAndRole(props) {
                   suggestedInstitution: value?.displayText
                 }));
               }
-            }
-          }),
-        ]
-      );
+            }}
+          >
+          </FormField>
+        </h>
+      </div>;
     } else {
       let institution = (profile.institutionId ? find(institutions, { id: profile.institutionId }) : null);
       const institutionName = (institution ? institution.name : profile.suggestedInstitution);
-
-
-      return div({
-        className: '',
-        style: { padding: 0 },
-      }, [
-        input({
-          id: 'profileInstitution',
-          name: 'institution',
-          type: 'text',
-          disabled: true,
-          className: 'form-control',
-          value: institutionName,
-        }),
-      ]);
+      return <div style={{ padding: 0 }}>
+        <input
+          id='profileInstitution'
+          name='institution'
+          type='text'
+          disabled={true}
+          className='form-control'
+          value={institutionName}
+        />
+      </div>;
     }
   };
 
-  return div({}, [
-    h1({
-      style: {
+  return <div>
+    <h1
+      style={{
         color: '#01549F',
         fontSize: '20px',
         fontWeight: '600',
-      }
-    }, ['Affiliation & Role']),
-    div({ className: '', style: { 'marginTop': '20px' } }, []),
-    div({
-      style:
-      {
+      }}
+    >
+      Affiliation & Role
+    </h1>
+    <div style={{ marginTop: '20px' }} />
+    <div
+      style={{
         color: '#000',
         fontFamily: 'Montserrat',
         fontSize: '16px',
         fontStyle: 'normal',
         fontWeight: '600',
         lineHeight: 'normal'
-      }
-    },
-    [
-      p({}, ['Institution']),
-      div({ style: { marginBottom: '15px' } }, []),
-      generateInstitutionSelectionDisplay(),
-      button({
-        id: 'btn_submit',
-        onClick: submitForm,
-        className: 'f-right btn-primary common-background',
-        style: {
+      }}
+    >
+      <p>Institution</p>
+      <div style={{ marginTop: '15px' }} />
+      {generateInstitutionSelectionDisplay()}
+      <button
+        id='btn_submit'
+        onClick={submitForm}
+        className='f-right btn-primary common-background'
+        style={{
           marginTop: '2rem',
-        },
-        disabled: !formIsValid(),
-      }, ['Save']),
-      div({ className: '', style: { 'marginTop': '83px' } }, []),
-      p({}, ['Role']),
-    ]),
-    p({}, [profile.roles]),
-  ]);
+        }}
+        disabled={!formIsValid()}
+      >
+        Save
+      </button>
```

Comment:
I think we could use some tool-tip text here to indicate why the save button is disabled. This is not in the stated requirements, but it would help reduce reliance on customer support.